lib/queries/source/autocomplete.rb

```diff
@@ -212,12 +212,11 @@ def autocomplete
           # order results by number of times used
           if scope
             a = a.left_outer_joins(:citations)
-              .select('sources.*, COUNT(citations.id) AS use_count, MAX(citations.project_id) AS in_project_id, project_sources.id AS project_source_id')
+              .select('sources.*, COUNT(citations.id) AS use_count, MAX(citations.project_id) AS in_project_id, MAX(project_sources.id) AS project_source_id')
```

Comment:
MAX doesn't seem right here, what is the goal?

Reply:
this does not do anything, practicing with sql and forgot to remove.

Reply:
I will remove.